src/e2e/java/teammates/e2e/cases/sql/FeedbackRankOptionQuestionE2ETest.java

```diff
@@ -69,20 +69,19 @@ protected void testEditPage() {
         verifyPresentInDatabase(copiedQuestion);
 
         ______TS("edit question");
-        questionDetails = (FeedbackRankOptionsQuestionDetails) loadedQuestion.getQuestionDetailsCopy();
         List<String> options = questionDetails.getOptions();
         options.remove(0);
         options.set(1, "Edited option.");
         questionDetails.setOptions(options);
         questionDetails.setAreDuplicatesAllowed(true);
         questionDetails.setMaxOptionsToBeRanked(Const.POINTS_NO_VALUE);
         questionDetails.setMinOptionsToBeRanked(1);
-        loadedQuestion.setQuestionDetails(questionDetails);
```

Comment:
could i check why this was changed from `loadedQuestion` to `copiedQuestion`? the previous test case edited the `loadedQuestion`

Reply:
Hey sorry for the late reply, when i use loaded question somehow it will consistently affect the testdata and the question order resulting in the testSubmitPage to fail. I have mentioned this to @weiquu as well. What do you think?